src/hooks/useEnsuredControl.ts

```diff
@@ -1,17 +1,57 @@
 import * as React from "react";
 
+interface UseEnsuredControlProps<V, E extends React.ChangeEvent<any>> {
+  value?: V;
+  onChange?: (v: E) => any;
```

Comment:
```suggestion
  onChange?(e: E): unknown;
```
+ 33 строчка

---

или на `unknown` ругаться будет?

Reply:
Будет ругаться, `v` поменял на `e`